scripts/msct_image.py

```diff
@@ -198,16 +198,40 @@ def decompose_affine_transform(A44):
 
 class Slicer(object):
     """
-    Image slicer utility class, that can help getting ranges and slice indices
+    Image(s) slicer utility class.
+
+    Can help getting ranges and slice indices.
```

Comment:
Yeah that was the mind-blowing part.

But note that the orientation of the remaining slice dimensions is not managed yet.
This will be done when the `Image` refactorings are in.